implementation/interfacing_with_emlib.tex

```diff
@@ -164,32 +164,226 @@ \subsection{Naming Conventions}
 \label{lst:enum_naming_rust}
 \end{listing}
 
+\subsection{The Bindings Libraries}
+\label{ssub:the_bindings_library}
+
+We have written many example applications in Rust that are based on bindings for many of the {\gecko}'s peripherals, these examples have been a driving factor for defining new bindings.
+As an example, if we were to need some functionality for the \gls{adc}, we would define these bindings alongside the development of the examples.
+In this way, the bindings have been defined incrementally during the development of either the examples that demonstrate our libraries, or the applications described in \todo{Refer to sections about circle-game and sensor-tracker}.
+
+\subsubsection{Emlib}
+
+It has not been a priority of this project to fully support all of the {\gecko}'s peripherals or provide bindings for the entire {\emlib}.
```

Comment:
Lets emphasis what was the priority more than what was not? E.g.

`The priority when making emlib bindings was to make a viable platform to create application which can be measured inorder to evaluate Rust in an embedded system`